portal/src/util/formatDatetime.ts

```diff
@@ -13,7 +13,7 @@ export function formatDatetime(
     datetime = DateTime.fromISO(date);
   }
 
-  if (datetime == null) {
+  if (!datetime?.isValid) {
```

Comment:
Can you clarify in what circumstances you receive an invalid date string from the server? We should fix the server instead of working it around here.

It is better to remove this change.

Reply:
server should return valid datetime string, will remove this